source/features/clean-issue-filters.tsx

```diff
@@ -4,11 +4,11 @@ import features from '../libs/features';
 import * as api from '../libs/api';
 import {getOwnerAndRepo} from '../libs/utils';
 
-interface CacheEntry {
+type CacheEntry = {
```

Comment:
👍

Reply:
The other issue I mentioned was solved/avoided by d94acb68a515a9e1bf5cfe8241837de495996c86